sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -406,6 +418,120 @@ The returned response is a heterogeneous list of result and error objects: list[
 Please refer to the service documentation for a conceptual discussion of [language detection][language_detection]
 and [language and regional support][language_and_regional_support].
 
+### Healthcare Analysis
+The example below extracts entities recognized within the healthcare domain, and identifies relationships between entities within the input document and links to known sources of information in various well known databases, such as UMLS, CHV, MSH, etc.  This sample demonstrates the usage for [long-running operations](#long-running-operations).
+
+```python
+from azure.core.credentials import AzureKeyCredential
+from azure.ai.textanalytics import TextAnalyticsClient
+
+credential = AzureKeyCredential("<api_key>")
+endpoint="https://<region>.api.cognitive.microsoft.com/"
+
+text_analytics_client = TextAnalyticsClient(endpoint, credential, api_version="v3.1-preview.3")
+
+documents = ["Subject is taking 100mg of ibuprofen twice daily"]
+
+poller = text_analytics_client.begin_analyze_healthcare(documents, show_stats=True)
+result = poller.result()
+
+docs = [doc for doc in result if not doc.is_error]
+
+print("Results of Healthcare Analysis:")
+for idx, doc in enumerate(docs):
+    for entity in doc.entities:
+        print("Entity: {}".format(entity.text))
+        print("...Category: {}".format(entity.category))
+        print("...Subcategory: {}".format(entity.subcategory))
+        print("...Offset: {}".format(entity.offset))
+        print("...Confidence score: {}".format(entity.confidence_score))
+        if entity.links is not None:
+            print("...Links:")
+            for link in entity.links:
+                print("......ID: {}".format(link.id))
+                print("......Data source: {}".format(link.data_source))
+    for relation in doc.relations:
+        print("Relation:")
+        print("...Source: {}".format(relation.source.text))
+        print("...Target: {}".format(relation.target.text))
+        print("...Type: {}".format(relation.relation_type))
+        print("...Bidirectional: {}".format(relation.is_bidirectional))
+    print("------------------------------------------")
+```
+
+The returned response is a heterogeneous list of result and error objects [[ MORE HERE ]]
```

Comment:
is this meant to have a link?

Reply:
Yep, meant to add but forgot.  I will add this now.  Thanks for catching!

Reply:
Actually, just removed this line for now since I'm not sure about the status of the generated docs.  I can make another update in the future as discussed earlier with @iscai-msft .